compiler/qsc_eval/src/val.rs

```diff
@@ -381,6 +381,31 @@ impl Value {
             Value::Var(_) => "Var",
         }
     }
+
+    #[must_use]
+    pub fn to_qubits(&self) -> Vec<Qubit> {
```

Comment:
The name of this made me think it's some kind of conversion operator, which confused me. It's more of a getter. `qubits()` would be a better name IMO (as in `get_qubits`, but Rust doesn't like prefixes in getter names)

Reply:
Updated the name and added a comment to help explain the usage.